machines/attic-server/configuration.nix

```diff
@@ -0,0 +1,23 @@
+{ self, ... }:
+{ pkgs, lib, config, ... }:
+{
+
+  networking = {
+    hostName = "attic-server";
+  };
+
+  mayniklas.user.root.enable = true;
+
+  lounge-rocks = {
+    # attic.server.enable = true;
+    nix-common.enable = true;
+    cloud-provider = {
+      enable = true;
+      proxmox.enable = true;
```

Comment:
Is a separate `enable` option really necessary? having just `proxxmox.enable` etc. would make sense

Reply:
How about we enable `cloud-provider.enable` within the provider specific configs?
This way we still have a general module for providers, that don't have any specific requirements.

Long time goal as I stated before: don't having much specifics.